src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -217,7 +217,9 @@ class BaseChildNode {
 // indirect mapping.
 template <typename T, typename node_key_t>
 inline BaseChildNode<T, node_key_t>* get_reserved_ptr() {
-  return (BaseChildNode<T, node_key_t>*)0x1;
+  //TODO: using instant integers as invalid pointers may
+  //	  not be a good practice.
+  return (BaseChildNode<T, node_key_t>*)0xdead0;
```

Comment:
Can we use a huge value close to the maximum instead?

That should make sure it is impossible to conflict with the normal ptr values.

Reply:
Done, please take a look.

Reply:
Probably better to calculate the value at the compile time rather than runtime?

Reply:
Ah, yes.